### PR TITLE
Bump to .NET 6 and .NET 8

### PR DESCRIPTION
**Pull request by flier268**

### CODE REVIEW

**Comment by TylerBrinkley** [comment]
Hi @flier268. For what reason does this need to target .NET 6 and .NET 8?

**Comment by flier268** [comment]
Alright, it seems that not compiling for .NET 6 and .NET 8 might be the
correct approach. However, while I was making changes, I noticed that .NET
8 has removed boolean enums. I'm wondering if this will have any impact?
[More on .NET 8 boolean enum changes](
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/8.0/boolean-backed-enum-type-support-removed
).

Tyler Brinkley ***@***.***> 於 2023年11月20日 週一 23:24 寫道：

**Comment by TylerBrinkley** [comment]
Thanks for the heads up. Yeah I think I probably need to follow suit since I'm using some System.Enum methods specifically for boolean enums due to a reflection bug. https://github.com/TylerBrinkley/Enums.NET/blob/a35664d000fb3e7c4f75ec303b8dc419837ab0d4/Src/Enums.NET/Enums.cs#L120-L121

File: Src/Enums.NET/Enums.NET.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;netcoreapp3.0;netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.1;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.1;netstandard1.0;net6.0;net8.0</TargetFrameworks>
```
Comment (thompson-tomo) [inline Src/Enums.NET/Enums.NET.csproj:3]: Because of the change in TFM the item groups below should be updated for an optimal dependency tree.

**Review by thompson-tomo — CHANGES_REQUESTED** [review]

**Comment by TylerBrinkley** [comment]
I bumped it to net 7 to incorporate the generic number interfaces.

**Pull request closed without merging**